lib/netutil/tls.go

```diff
@@ -8,10 +8,24 @@ import (
 	"sync"
 
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/fasttime"
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/fs"
 )
 
 // GetServerTLSConfig returns TLS config for the server.
 func GetServerTLSConfig(tlsCertFile, tlsKeyFile, tlsMinVersion string, tlsCipherSuites []string) (*tls.Config, error) {
+	if !fs.IsPathExist(tlsCertFile) {
+		return nil, fmt.Errorf("cannot find TLS cert file: %q", tlsCertFile)
+	}
+
+	if !fs.IsPathExist(tlsKeyFile) {
+		return nil, fmt.Errorf("cannot find TLS key file: %q", tlsKeyFile)
+	}
```

Comment:
Would ` tls.LoadX509KeyPair` make these checks obsolete?

Reply:
yep, just use tls.LoadX509KeyPair is a better way.